### PR TITLE
chore(pinbot): update README.md

### PR DESCRIPTION
**Pull request by simonovic86**
- update `README.md` for spinning Ghost Pinbot server

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,118 @@
...
+
+In order to setup secure WebSocket for `swarm` direct connect over `https` that's automatically implemented in the Terraform scripts provided. The only
+input that needs to be provided is the certificate arn from AWS.
+
+If you want to setup secure WebSocket for `swarm` without using AWS and the scripts provided, feel free to set it up using `nginx` or 
+other software to create a reverse proxy.
+
+## Swarm
+
+Upon successful start and deployment, the MultiAddress of a Ghost Pinbot node can be passed as an option `ghostPinbot` to `3Box` on create.
```
Comment (oed) [inline README.md:34]: Maybe add a sentence of how you can get the multiaddress of the pinbot.

**Review by oed — COMMENTED** [review]
This looks great, just one addition I'd like to see!

File: README.md (hunk 1 above)
Comment (simonovic86) [reply README.md:34]: Makes sense 👍  I added a couple of utility routes

**Review by oed — APPROVED** [review]
Looks good, but what's up with code indention?

**Comment by simonovic86** [comment]
> Looks good, but what's up with code indention?

that should be fixed now :) I merged it with `master` where linting is enabled

**Pull request merged**